docs/cli_interface.md

```diff
@@ -55,12 +55,12 @@ Sentiment Treebank, a set of sentiment tags over English reviews.
 
 The below example loads the `sst2` dataset from DataLab:
 ```shell
-explainaboard --task text-classification --dataset sst2 --system-outputs ./data/system_outputs/sst2/sst2-lstm-output.txt
+explainaboard --task text-classification --dataset sst2 --system_outputs ./data/system_outputs/sst2/sst2-lstm-output.txt
```

Comment:
It looks this change is wrong. We follow the GNU style for every CLI options.

https://github.com/neulab/ExplainaBoard/blob/ca23011603db2b6cfda468ba201af4dec74aaf97/explainaboard/explainaboard_main.py#L139-L148

Reply:
Seems weird because when I run `explainaboard --task text-classification --dataset sst2 --system_outputs ./data/system_outputs/sst2/sst2-lstm-output.txt` (with underscore) it works. But when I run `explainaboard --task text-classification --dataset sst2 --system-outputs ./data/system_outputs/sst2/sst2-lstm-output.txt`, I get

```
usage: explainaboard [-h] [--task TASK] --system_outputs SYSTEM_OUTPUTS [SYSTEM_OUTPUTS ...] [--reports REPORTS [REPORTS ...]] [--systems SYSTEMS [SYSTEMS ...]] [--datasets DATASETS [DATASETS ...]] [--languages LANGUAGES [LANGUAGES ...]]
                     [--systems_aggregation SYSTEMS_AGGREGATION] [--datasets_aggregation DATASETS_AGGREGATION] [--languages_aggregation LANGUAGES_AGGREGATION] [--dataset DATASET] [--sub_dataset SUB_DATASET] [--split SPLIT] [--language TARGET_LANGUAGE]
                     [--source_language SOURCE_LANGUAGE] [--reload_stat RELOAD_STAT] [--metrics [METRICS [METRICS ...]]] [--output_file_type OUTPUT_FILE_TYPE] [--conf_value CONF_VALUE] [--output_dir OUTPUT_DIR] [--report_json REPORT_JSON] [--system_details SYSTEM_DETAILS]
                     [--custom_dataset_paths [CUSTOM_DATASET_PATHS [CUSTOM_DATASET_PATHS ...]]] [--custom_dataset_file_type CUSTOM_DATASET_FILE_TYPE]
explainaboard: error: the following arguments are required: --system_outputs
```
It somehow says it requires `--system_outputs`.

This seems a arg parser problem rather than doc problem. Will revert the change on the doc and we can fix the problem.

Reply:
It looks the version of the installed binary on your machine is old for some reason. Could you try to re-install it?